### ISSUE TITLE
Several issues

### ISSUE THREAD

**Issue by hakase420** [OpenTabletDriver/OpenTabletDriver#1431]
Hello everybody, I have several issues. It all started a couple of hours ago where I wanted to play Osu but it didn't work. At the time I used TabletdriverGUI. So i hopped on the github and searched for my issue, because in the console it said there was a read error. Some dudes already asked for help and they all got a link to a other post, where it said that you need to use OpenTabletDriver instead. Then I installed this and it can't find my tablet. So once again I searched for it and it said that I need to delete every other tablet driver, which I did. But it still doesn't find my damn tablet. I have a Huion H640P and it should be supported. I don't really know what to do anymore. I don't care which tablet driver I use, so if you only have a solution for one of the problems, it's also good. 

tl;dr Couple of hours ago TabletdriverGUI stopped working for some reason, I installed OpenTabletDriver because github said so, but it can't find my tablet, altough I deleted TabletdriverGUI and my tablet a Huion H640P should be supported.

**Comment by X9VoiD** [comment]
Make sure you've already uninstalled Huion driver. If it is already uninstalled then follow instructions from here:

https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Windows-FAQ#my-gaomonhuion-tablet-isnt-detected-but-is-listed-as-supported

Closing this issue as this is not an issue within OTD.
